v2/samples/dataprotection/v1api/v1api20230101_backupvault.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: dataprotection.azure.com/v1api20230101
+kind: BackupVault
+metadata:
+  name: asotestbackupvault
+  namespace: default
+spec: 
+  location: eastus
+  owner:
+    name: aso-sample-rg
+  identity:
+    type: SystemAssigned
+  tags:
+    tag1: value1
+  properties:
+    monitorsettings:
```

Comment:
I think that your sample casing here is incorrect, and the actual name of this field is also incorrect?

The type has:
`MonitoringSettings *MonitoringSettings `json:"monitoringSettings,omitempty"``

So (AFAIK) this should be `monitoringSettings:`

The same applies to the casing of other parts of this file as well. They all must be correctly cased to match the spec in the code.

Reply:
Yes, the sample casing hasn't been addressed properly before. Thanks for noticing. Rectified in all ample Yaml files.